src/constants/staticEndpoints.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   [apiReqNames.GET_USER]: fullEndpointForUserId,
   [apiReqNames.PATCH_ADD_NEW_DOCUMENTS]: fullEndpointForUserId + '?full_dehydrate=yes',
   [apiReqNames.PATCH_UPDATE_DOCUMENTS]: fullEndpointForUserId + '?full_dehydrate=yes',
+  [apiReqNames.PATCH_UPDATE_REG_GG]: fullEndpointForUserId, // response doesn't change w full_dehydrate
```

Comment:
This endpoint should like this: `https://uat-api.synapsefi.com/v3.1/users/user_id/ubo`

ref: https://docs.synapsefi.com/v3.1.1/reference#generate-ubo-reggg

Reply:
whoops mistake, updated